packages/theming/atlas/tsconfig.json

```diff
@@ -1,12 +1,9 @@
 {
     "extends": "../../tools/pluggable-widgets-tools/configs/tsconfig.base",
-    "include": [
-      "./src/native/**/*.ts"
-    ],
+    "include": ["./src/theme/native/**/*.ts", "./src/themesource/atlas_ui_resources/native/**/*.ts"],
     "compilerOptions": {
         "target": "ES2019",
         "lib": ["ES2017"],
-        "sourceMap": false,
-        "outDir": "./dist/theme/styles/native"
```

Comment:
without `outDir` an attempt to run `tsc` (e.g. by your ide) will produce js files next to ts sources, which can get messy.

Reply:
My reasoning was that there is no value in having a `dist/` folder anymore. We didn't use the output in there and probably never will because of the modularisation. 
In our build script we do add the `outDir` parameter, which targets a mendix project.
I don't think anyone would want to run `tsc` without parameters as the output has to be selectively copied over to a mendix project anyhow which is way more work then just running the build script :)

Reply:
I've been few times in a situation where i run "compile typescript" from my ide just to make sure there are no errors.

Reply:
But it is up to you :)

Reply:
That makes sense, I've added it again :)